apps/08jquery-towers-of-hanoi/script.js

```diff
@@ -1,5 +1,49 @@
 'use strict';
 
 $(document).ready(function() {
-  // Put app logic here
+
+  var $dataStack = $("[data-stack]");
+  var block = null;
+
+  // When the player manipulates the stack by clicking on a block, remove the last block(child) from the stack. Then, drop the block on a seperate line or stack, that is clicked.
+  $dataStack.click(function() {
+
+    if (block === null) {
+
+      var child = $(this).children();
+
+      var last = child.last();
+
+      block = last.detach();
+
+      return true;
+    } else {
+      // Checks to see if the block you are moving is smaller than the block you are moving it to. If not, It ain't gonna work!
+      var $lastBlock = $(this).children().last().data('block');
+
+      var $blockToPlace = block.data('block');
+
+      if (Number($lastBlock) < Number($blockToPlace)) {
```

Comment:
Nice use of Number(). It doesn't appear to be 100% necessary here, but it doesn't hurt! A comment here  explaining what Number is doing would be awesome